frontend/tests/components/droplets/enroll-button.test.tsx

```
@@ -3,6 +3,7 @@ import { EnrollButton } from "@/components/droplets/enroll-button";
 import { useRouter } from "next/navigation";
 import { toast } from "sonner";
 import { createEnrollment, deleteEnrollment } from "@/lib/requests/enrollment";
+import { drop } from "lodash";
```

ESLint

[error] 'drop' is defined but never used.

(@typescript-eslint/no-unused-vars)